helix-rest/src/main/java/org/apache/helix/rest/server/resources/metadatastore/MetadataStoreDirectoryAccessor.java

```diff
@@ -65,6 +66,11 @@ private void postConstruct() {
     buildMetadataStoreDirectory(_namespace, helixRestNamespace.getMetadataStoreAddress());
   }
 
+  @PreDestroy
```

Comment:
Let's try to have changes like this in a separate PR. Or you could create an issue and we could batch process them later.

Reply:
Forgot to remove it. Not supposed to include it.